daxctl: add libuuid to the build

BZ: 

commit 22bffbecb16d2aa7ed072fb55a18445977ee16b8
Author: Dan Williams <dan.j.williams@intel.com>
Date:   Mon Feb 13 21:01:25 2017 -0800

    daxctl: add libuuid to the build
    
    daxctl does have dependencies on the uuid library via util/json.c, but
    the build only seems to fail in a Debian build environment. Fix this up.
    
    Reported-by: Yilong Ren <yilongx.ren@intel.com>
    Signed-off-by: Dan Williams <dan.j.williams@intel.com>

diff --git a/daxctl/Makefile.am b/daxctl/Makefile.am
index 9153c41..fe467d0 100644
--- a/daxctl/Makefile.am
+++ b/daxctl/Makefile.am
@@ -10,4 +10,5 @@ daxctl_SOURCES =\
 daxctl_LDADD =\
 	lib/libdaxctl.la \
 	../libutil.a \
+	$(UUID_LIBS) \
 	$(JSON_LIBS)
